KMSKeyArn - nil pointer segmentation

- darwin_amd64
  - master branch, at commit 23404a9:

This is a regression due to #648 

If KMSKeyArn parameter is not set il function.json we have a segmentation due to nil pointer dereference


```
➜  apex git:(master) ✗ go run cmd/apex/main.go -e production  -C ~/Workspace/service-checks deploy --dry-run

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x1f91aa]

goroutine 19 [running]:
panic(0x5f2a80, 0xc4200140b0)
	/usr/local/Cellar/go/1.7.5/libexec/src/runtime/panic.go:500 +0x1a1
github.com/apex/apex/function.(*Function).configChanged(0xc4201ec000, 0xc4201542c0, 0x0)
	/Users/gvisciano/Workspace/Personal/go/src/github.com/apex/apex/function/function.go:785 +0x32a
github.com/apex/apex/function.(*Function).Deploy(0xc4201ec000, 0x703d40, 0xc4201e22a0)
	/Users/gvisciano/Workspace/Personal/go/src/github.com/apex/apex/function/function.go:253 +0x164
github.com/apex/apex/project.(*Project).Deploy.func1.1(0xc4201e22a0, 0xc4201ec000, 0xc4201e2300)
	/Users/gvisciano/Workspace/Personal/go/src/github.com/apex/apex/project/project.go:203 +0x6f
created by github.com/apex/apex/project.(*Project).Deploy.func1
	/Users/gvisciano/Workspace/Personal/go/src/github.com/apex/apex/project/project.go:209 +0xb0
exit status 2
```

--- a/function/function.go
+++ b/function/function.go
@@ -782,7 +782,10 @@ func (f *Function) configChanged(config *lambda.GetFunctionOutput) bool {
 		Role:        *config.Configuration.Role,
 		Runtime:     *config.Configuration.Runtime,
 		Handler:     *config.Configuration.Handler,
-		KMSKeyArn:   *config.Configuration.KMSKeyArn,
+	}
+
+	if config.Configuration.KMSKeyArn != nil {
+		remoteConfig.KMSKeyArn = *config.Configuration.KMSKeyArn
 	}
 
 	if config.Configuration.Environment != nil {

